spec/asyncapi.md

```diff
@@ -1190,7 +1256,7 @@ The following table contains a set of values that every implementation is RECOMM
 Name | Allowed values | Notes
 ---|:---:|---
 [Avro 1.9.0 schema](https://avro.apache.org/docs/1.9.0/spec.html#schemas) | `application/vnd.apache.avro;version=1.9.0`, `application/vnd.apache.avro+json;version=1.9.0`, `application/vnd.apache.avro+yaml;version=1.9.0` |
-[OpenAPI 3.0.0 Schema Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schemaObject) | `application/vnd.oai.openapi;version=3.0.0`, `application/vnd.oai.openapi+json;version=3.0.0`, `application/vnd.oai.openapi+yaml;version=3.0.0` | 
+[OpenAPI 2.5.0 Schema Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.5.0.md#schemaObject) | `application/vnd.oai.openapi;version=2.5.0`, `application/vnd.oai.openapi+json;version=2.5.0`, `application/vnd.oai.openapi+yaml;version=2.5.0` | 
```

Comment:
You should revert this change. This is about OpenAPI 3.0 not AsyncAPI :)

Reply:
ops was fixed